src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-node/psbt-unsigned-input-list/components/psbt-unsigned-input-item.tsx

```diff
@@ -12,33 +14,34 @@ import { PsbtUnsignedInputWithInscription } from './psbt-unsigned-input-with-ins
 interface PsbtUnsignedInputItemProps {
   addressNativeSegwit: string;
   addressTaproot: string;
-  utxo: TaprootUtxo & OrdApiInscriptionTxOutput;
+  utxo: BitcoinTransactionVectorOutput;
+  utxoWithInscription: (TaprootUtxo & OrdApiInscriptionTxOutput) | undefined;
 }
 export function PsbtUnsignedInputItem({
   addressNativeSegwit,
   addressTaproot,
   utxo,
+  utxoWithInscription,
 }: PsbtUnsignedInputItemProps) {
   const calculateBitcoinFiatValue = useCalculateBitcoinFiatValue();
 
   const isInputCurrentAddress =
-    utxo.address === addressNativeSegwit || utxo.address === addressTaproot;
+    utxo.scriptpubkey_address === addressNativeSegwit ||
+    utxo.scriptpubkey_address === addressTaproot;
   const inputValue = satToBtc(utxo.value).toString();
-  const fiatValue = i18nFormatCurrency(calculateBitcoinFiatValue(utxo.value));
-  const inscription = utxo.inscriptions;
-
-  if (!utxo.address) return null;
+  const fiatValue = i18nFormatCurrency(calculateBitcoinFiatValue(inputValue));
+  const inscription = utxoWithInscription?.inscriptions;
 
-  return inscription ? (
+  return inscription && utxoWithInscription.address ? (
     <PsbtUnsignedInputWithInscription
-      address={utxo.address}
+      address={utxoWithInscription.address}
       inputValue={inputValue}
       path={inscription}
     />
   ) : (
     <PsbtDecodedNodeLayout
-      hoverLabel={utxo.address}
-      subtitle={truncateMiddle(utxo.address)}
+      hoverLabel={utxo.scriptpubkey_address}
+      subtitle={truncateMiddle(utxo.scriptpubkey_address)}
       subValue={`${fiatValue} USD`}
```

Comment:
This makes `$10.00 USD`. Do we want both the `$` and the 4217 currency code?

Reply:
I think this is a design question? @mica000 pretty sure this is on the designs? I'll double check.

Reply:
Opsi, lets use only $ to keep consistent with how we show on the homepage?